app/src/main/java/com/imashnake/animite/features/theme/Type.kt

```diff
@@ -17,3 +20,42 @@ val manropeFamily = FontFamily(
     Font(R.font.manrope_light, FontWeight.Light),
     Font(R.font.manrope_extralight, FontWeight.ExtraLight)
 )
+
+// TODO: Label these!
+val AnimiteTypography = Typography(
```

Comment:
If you're wanting to follow Material Design, I'd suggest using font sizes which match the typography spec - https://material.io/design/typography/the-type-system.html#type-scale

Reply:
I'll address this in #48.